moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -253,9 +253,7 @@ const ProjectFundingTable = () => {
       PopperComponent={CustomPopper}
       id={props.name}
       options={props.data}
-      renderInput={(params) => (
-        <TextField variant="standard" {...params} multiline />
```

Comment:
I removed multiline because there was a warning that shows up
```
ProjectFundingTable.js:649 A textarea element was provided to Autocomplete where input was expected.
This is not a supported scenario but it may work under certain conditions.
A textarea keyboard navigation may conflict with Autocomplete controls (for example enter and arrow keys).
Make sure to test keyboard navigation and add custom event handlers if necessary. 
```

And I dont think the autocompletes need to be multiline for this table

Reply:
Thanks for addressing this! 🙏 I agree that multiline is not needed since the chosen source or program is highlighted in the dropdown if someone needs to double-check their selection.